agents/insurance-solution/agent/lambda/action-groups/create_claim.py

```diff
@@ -0,0 +1,103 @@
+import os
+import io
+import re
+import json
+import time
+import boto3
+import base64
+import random
+import string
+import decimal
+import requests
+
+# DynamoDB boto3 resource and variable
+dynamodb = boto3.resource('dynamodb',region_name=os.environ['AWS_REGION'])
+existing_claims_table_name = os.environ['EXISTING_CLAIMS_TABLE_NAME']
+
+# SNS boto3 clients and variables
+sns_topic_arn = os.environ['SNS_TOPIC_ARN']
+sns_client = boto3.client('sns')
+
```

Comment:
Could you add the support functionalities to read the parameters from GET and POST requests? We want to promote them as easy to use helper functions..

```
def get_named_parameter(event, name):
    return next(item for item in event['parameters'] if item['name'] == name)['value']

def get_named_property(event, name):
    return next(item for item in event['requestBody']['content']['application/json']['properties'] if item['name'] == name)['value']
```